contracts/modules/RecoveryManager.sol

```diff
@@ -115,12 +157,12 @@ contract RecoveryManager is BaseModule, RelayerModule {
      * @param _wallet The target wallet.
      */
     function finalizeRecovery(BaseWallet _wallet) external onlyExecute onlyWhenRecovery(_wallet) {
```

Comment:
We can remove the `onlyExecute` modifier. Is is part of another PR but it makes more sense to have it here.